platform/ios/test/MGLNetworkConfigurationTests.mm

```diff
@@ -40,4 +41,21 @@ - (void)testAccessingEventsFromMultipleThreads {
     
     [self waitForExpectations:@[expectation] timeout:10.0];
 }
+
+- (void)testStopsRequests {
+    auto networkStatus = mbgl::NetworkStatus::Get();
+    BOOL offline = networkStatus == mbgl::NetworkStatus::Status::Offline ? YES : NO;
+    XCTAssertEqual([MGLNetworkConfiguration sharedManager].stopsRequests, offline);
+    
+    [MGLNetworkConfiguration sharedManager].stopsRequests = YES;
+    XCTAssertTrue([MGLNetworkConfiguration sharedManager].stopsRequests);
+    
+    networkStatus = mbgl::NetworkStatus::Get();
+    offline = networkStatus == mbgl::NetworkStatus::Status::Offline ? YES : NO;
+    XCTAssertEqual([MGLNetworkConfiguration sharedManager].stopsRequests, offline);
+    
+    networkStatus = mbgl::NetworkStatus::Get();
+    offline = networkStatus == mbgl::NetworkStatus::Status::Offline ? YES : NO;
+    XCTAssertEqual([MGLNetworkConfiguration sharedManager].stopsRequests, offline);
+}
```

Comment:
These tests look good, thanks! Have you considered implementing tests of the offline/network functionality — e.g., does enabling this property actually result in no network requests?

Reply:
My understanding is that this will stop tile and related requests, but other network activity would still occur.